src/server/routes/changePassword.ts

```diff
@@ -167,6 +167,7 @@ router.get(
   (_: Request, res: ResponseWithLocals) => {
     const html = renderer(Routes.RESET_RESEND, {
       pageTitle: PageTitle.RESET_RESEND,
+      locals: res.locals,
```

Comment:
we need to propagate res.locals for all routes because the CSRF token is set on locals